ta/arch/arm/link.mk

```diff
@@ -72,12 +82,21 @@ $(link-out-dir$(sm))/$(user-ta-uuid).stripped.elf: \
 	@$(cmd-echo-silent) '  OBJCOPY $$@'
 	$(q)$(OBJCOPY$(sm)) --strip-unneeded $$< $$@
 
+ifeq ($(CFG_ENCRYPT_TA),y)
```

Comment:
Not the best condition IMO. This means: "Is OP-TEE capable of loading encrypted TAs", which should be different from "Should this TA be encrypted". Maybe test `ifneq (,$(TA_ENC_KEY))` instead, this way it is very easy to set or unset it on a per-TA basis.

Reply:
> This means: "Is OP-TEE capable of loading encrypted TAs"

`CFG_ENCRYPT_TA` isn't an OP-TEE OS configuration option but rather a TA build time option to provide answer to "Should this TA be encrypted". Also it can be enabled on per-TA basis.